packages/common/src/hooks/useFeatureFlag.ts

```diff
@@ -146,25 +146,25 @@ export const useFeatureFlag = (
     [overrideKey, localStorage]
   )
 
-  const [isLocallyEnabled, setIsLocallyOverriden] = useState<Maybe<boolean>>()
+  const [isLocallyEnabled, setIsLocallyOverriden] =
+    useState<Maybe<Nullable<boolean>>>()
 
   useEffectOnce(() => {
     const getOverride = async () => {
       const override = await localStorage.getItem(overrideKey)
       if (override === 'enabled') {
         setIsLocallyOverriden(true)
-      }
-      if (override === 'disabled') {
+      } else if (override === 'disabled') {
         setIsLocallyOverriden(false)
+      } else {
+        setIsLocallyOverriden(null)
       }
-
-      return undefined
```

Comment:
I find myself wondering why this was here in the first place? Seems like it was meant to make sure the effect got undefined as a return value, but it doesn't seem like this return value was ever used.